examples/bunny-twirl/index.js

```diff
@@ -56,13 +56,26 @@ container.addChild(text)
 let isAnimating = true
 let isPlaying = false
 
+// Generate map
+const challengeNumber = Rune.getChallengeNumber();
+// const nextChallengeNumber = challengeNumber + 1;
+
+// 1. Game map randomization
```

Comment:
I realize this is aligned with the README section title, but it seems odd in the context of this game. Should we add something like `// Derive rotation speed (floating-point number between 0.005 and 0.03) from Rune challenge number` to explain what we're actually doing?

Reply:
Fixed in https://github.com/rune/rune-games-sdk/pull/24/commits/75d27bf8f857bf99a576dc5fd7abcb06909a0d51